src/.jshintrc

```diff
@@ -12,6 +12,9 @@
 
 	"sub": true,
 
+	// Support: IE < 10, Android < 4.1
+	// The above browsers are failing a lot of tests in the ES5
+	// test suite at http://test262.ecmascript.org.
```

Comment:
I didn't realize it's undocumented (I assume it should be, @rwaldron?). I agree it feels bad, it just felt more bad to me to have this undocumented and have someone remove the option prematurely, as we did on `master` before.